content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -974,6 +976,12 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `LegacyNodeRoleBehavior`: When disabled, legacy behavior in service load balancers and
   node disruption will ignore the `node-role.kubernetes.io/master` label in favor of the
   feature-specific labels provided by `NodeDisruptionExclusion` and `ServiceNodeExclusion`.
+- `KubeletTracing`: Add support for distributed tracing in the kubelet.
+  When enabled, kubelet CRI interface and authenticated http servers are instrumented to generate
+  OpenTelemetry trace spans. Sampling rate and collector address are configurable. When enabled without
+  a tracing configuration, a parent span's sampling decision is respected and the default collector
+  address of 0.0.0.0:4317 is set.
```

Comment:
Please move:
> Sampling rate and collector address are configurable. When enabled without
  a tracing configuration, a parent span's sampling decision is respected and the default collector
  address of 0.0.0.0:4317 is set.

into the concept docs.

Reply:
done!